extensions/typescript/package.json

```diff
@@ -7,6 +7,7 @@
   "license": "MIT",
   "publisher": "vscode",
   "aiKey": "AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217",
+  "enableProposedApi": true,
```

Comment:
My fault. I moved the API but I need to remove a wrapper function. Will do an push.